layouts/_default/single.html

```diff
@@ -39,7 +39,7 @@
         <div class="author-profile ml-auto align-self-lg-center">
           <img class="rounded-circle" src='{{ partial "helpers/get-author-image.html" . }}' alt="Author Image">
           <h5 class="author-name">{{ partial "helpers/get-author-name.html" . }}</h5>
-          <p class="text-muted">{{ .Page.Date | time.Format ":date_full" }}{{ if site.Params.features.readingTime }} | {{ .ReadingTime }} minute{{ if (ne .ReadingTime 1) }}s{{ end }}{{ end }}</p>
+          <p class="text-muted" style="text-align: center;">{{ .Page.Date | time.Format ":date_full" }}{{ if site.Params.features.readingTime }} | {{ .ReadingTime }} minute{{ if (ne .ReadingTime 1) }}s{{ end }}{{ end }}</p>
```

Comment:
Please don't use inline css. Just add a `p` element under `author-profile` and apply CSS there:
https://github.com/hugo-toha/toha/blob/5b31653ed24d1607edbfe0bb598a381b72bf4deb/assets/styles/layouts/single.scss#L42

Reply:
Ok, now I added the text-align into the css file.